src/e2e/java/teammates/e2e/cases/InstructorHomePageE2ETest.java

```diff
@@ -61,6 +62,15 @@ protected void prepareTestData() {
     @BeforeClass
     public void classSetup() {
         deleteDownloadsFile(fileName);
+
+        // ------------------------------------ Prepare student non-submitter data ------------------------------------ //
+
+        StudentAttributes secondStudentNonSubmitter = testData.students.get("IHome.benny.c.tmms@IHome.CS2104");
+        secondStudentNonSubmitter.setEmail(TestProperties.TEST_EMAIL);
+        StudentAttributes thirdStudentNonSubmitter = testData.students.get("IHome.danny.e.tmms@IHome.CS2104");
+        thirdStudentNonSubmitter.setEmail(TestProperties.TEST_EMAIL);
+
+        studentNonSubmitters = Arrays.asList(studentToEmail, secondStudentNonSubmitter, thirdStudentNonSubmitter);
```

Comment:
I think it would be more appropriate to place this in the `prepareTestData` method, since this seems like preparing test data.

Reply:
I thought so too, but saw that `InstructorFeedbackReportPageE2ETest.java`, was doing it in the `classSetup()`, to keep the `prepareTestData` method lean

Reply:
Yes! For `InstructorFeedbackReportPageE2ETest.java`, in the `classSetup()` method, they are fetching data from the database but not modifying anything. In the `prepareTestData` method, they are modifying the testData that will be persisted into the database. In fact, there is a similar line of code in `prepareTestData()` that is almost identical to the changes you made.

```
studentToEmail = testData.students.get("Emily");
studentToEmail.setEmail(TestProperties.TEST_EMAIL);
removeAndRestoreDataBundle(testData);
```

Reply:
Yea I saw this, I thought because the `studentToEmail` and the `instructor` are entities used in every test as compared to my test data which was used only in one test, I would have to differentiate them by placing them in a different method. Will make the changes!